picker_wrapper.go

```diff
@@ -125,6 +130,10 @@ func (pw *pickerWrapper) pick(ctx context.Context, failfast bool, info balancer.
 					return nil, balancer.PickResult{}, status.Error(codes.Canceled, errStr)
 				}
 			case <-ch:
+				for _, sh := range pw.statsHandlers {
```

Comment:
Maybe a comment here or where `statsHandler` field is defined about what this is for and how this is used?

Reply:
Added comment on field: statsHandlers []stats.Handler // to record blocking picker calls